api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -79,15 +78,17 @@ message ListenersConfigDump {
 // configuration information can be used to recreate an Envoy configuration by populating all
 // clusters as static clusters or by returning them in a CDS response.
 message ClustersConfigDump {
-  // This is the *version_info* in the last processed CDS discovery response. If there
-  // are only static bootstrap clusters, this field will be "".
+  // This is the :ref:`version_info <envoy_api_field_DiscoveryResponse.version_info>` in the
```

Comment:
Is EDS supported? This would be needed for a complete dump I think, right?

Reply:
It's not supported currently. Host information is available via the /clusters endpoint and how EDS is handled via the config tracker interface is not super simple. I would rather have someone do this in a follow up if this is important to them. I don't plan on doing this as part of this change. Happy to document that somewhere if you like.

Reply:
If you can add some TODOs and file an issue, that fine. Maybe also change the PR description to indicate this is mostly there but not EDS yet.